File: Unidad 4/C#/P15_U4.cs
using System;
class P15_U4 {
  static void Main() {
            Console.Write("Ingrese el número de filas: ");
            int filas = Convert.ToInt32(Console.ReadLine());

            Console.Write("Ingrese el número de columnas: ");
            int columnas = Convert.ToInt32(Console.ReadLine());

            double[,] matriz = new double[filas, columnas];

            Console.WriteLine("Ingrese los elementos de la matriz:");

            for (int i = 0; i < filas; i++)
            {
                for (int j = 0; j < columnas; j++)
                {
                    Console.Write($"Elemento [{i + 1}][{j + 1}]: ");
                    matriz[i, j] = Convert.ToDouble(Console.ReadLine());
                }
            }

            Console.WriteLine("\nMedia de cada fila:");

            for (int i = 0; i < filas; i++)
            {
                double suma = 0;

                for (int j = 0; j < columnas; j++)
                {
                    suma += matriz[i, j];
                }

                double media = suma / columnas;
                Console.WriteLine($"Fila {i + 1}: {media}");
            }
        }
    }

File: Unidad 4/JAVA/P15_U4.java
public class Main
{
	public static void main(String[] args) {
	Scanner scanner = new Scanner(System.in);
        System.out.print("Ingrese el número de filas: ");
        int filas = scanner.nextInt();
        System.out.print("Ingrese el número de columnas: ");
        int columnas = scanner.nextInt();
        double[][] matriz = new double[filas][columnas];
        System.out.println("Ingrese los elementos de la matriz:");
        for (int i = 0; i < filas; i++) {
            for (int j = 0; j < columnas; j++) {
                System.out.print("Elemento [" + (i + 1) + "][" + (j + 1) + "]: ");
                matriz[i][j] = scanner.nextDouble();
            }
        }
        System.out.println("\nMedia de cada fila:");
        for (int i = 0; i < filas; i++) {
            double suma = 0;
            for (int j = 0; j < columnas; j++) {
                suma += matriz[i][j];
            }
            double media = suma / columnas;
            System.out.println("Fila " + (i + 1) + ": " + media);
        }
        scanner.close();
    }
}